Client/qtTeamTalk/mainwindow.ui

```diff
@@ -1872,7 +1872,7 @@
   </action>
   <action name="actionStoreForMove">
    <property name="text">
-    <string>&amp;Store User(s) for Move</string>
```

Comment:
"Store" is more correct because the user is added to a list that is emptied later